### PR TITLE
Updated Scala from 2.11.4 to 2.11.5

### PR DESCRIPTION
**Pull request by rmuhamedgaliev**
Updated scala version.

### CODE REVIEW

File: Library/Formula/scala.rb [hunk 1]
```diff
@@ -64,3 +64,4 @@ def main(args: Array[String]) {
     assert_equal "4", shell_output("#{bin}/scala #{file}").strip
   end
 end
+
```
Comment (mikemcquaid) [inline Library/Formula/scala.rb]: :question:

File: Library/Formula/scala.rb [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
   homepage 'http://www.scala-lang.org/'
-  url 'http://www.scala-lang.org/files/archive/scala-2.11.4.tgz'
-  sha1 'a6d319b26ccabe9c609fadebc32e797bf9cb1084'
+  url 'http://www.scala-lang.org/files/archive/scala-2.11.5.tgz'
+  sha1 '10820b7d65727fe8d3d67855ef603c425cf65a4e'
 
   bottle do
     cellar :any
-    sha1 "3135e56649f81649a90ef0cddb3fa9c9a8208864" => :yosemite
+    sha1 "888353d0e10f6449c97439677c8a820f408d9e71" => :yosemite
```
Comment (mikemcquaid) [inline Library/Formula/scala.rb]: Don't change these, we'll update when we pull.

**Comment by rmuhamedgaliev** [comment]
I сorrected according to the comments

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! For future reference the preferred commit message format for simple version updates is `scala 2.11.5`.

**Pull request closed without merging**